### ISSUE TITLE
Code in sandbox doesn't seem to work

### ISSUE THREAD

**Issue by jsduffy** [maxfahl/Quill-Edit-Multiple#1]
Thanks for posting this (we're looking for a similar fix).  Looks like the code in your sandbox doesn't work for me... when I click on a section I can't edit.

**Comment by maxfahl** [comment]
Hey! Are you testing it [here](https://codesandbox.io/s/mutable-breeze-h4cgs)?

It's working fine for me as of right now, using Chrome. Are you double clicking the column?

**Comment by jsduffy** [comment]
yeah, sorry I'm an idiot and I wasn't double clicking.  I'm trying to make something similar but need to allow the users to click directly into the editor at the position where they want to start editing with a single click.  Thanks for posting this though, it's been useful in trying to wade through Quills event handling.

**Comment by mcd92** [comment]
@jsduffy Hi, did you manage to change the code to achieve what you mentioned? (a single click to get both to focus in the editor and place the cursor in the clicked position)
